### DISCUSSION TITLE
迅雷云盘不支持webdav吗？

### DISCUSSION THREAD

**Discussion by xiaoxiyao** [AlistGo/alist#6956] [Q&A]
挂载的迅雷专家版，网页端可正常播放，webdav无法播放，试过kodi、potplayer、filmly都不行，看文档中webdav支持的存储类型中也没有迅雷，是无法支持吗？
[webdav-存储支持](https://alist.nn.ci/zh/guide/webdav.html#webdav-%E5%AD%98%E5%82%A8%E6%94%AF%E6%8C%81)

**Comment by xiaoxiyao** [comment]
经测试，启用视频url，或者将webdav策略改为本地代理，就可以播放了。
但启用视频url后，拖动进度条画面会卡住，声音还在正常播放

**Reply by 1109wuwuwu** [reply]
专业版用的302可以和客户端一个速度吗？有svip
